packages/airnode-utilities/src/evm/gas-prices/gas-oracle.ts

```diff
@@ -257,6 +281,8 @@ export const attemptGasOracleStrategy = (
       return fetchLatestBlockPercentileGasPrice(provider, gasOracleConfig, startTime);
     case 'providerRecommendedGasPrice':
       return fetchProviderRecommendedGasPrice(provider, gasOracleConfig, startTime);
+    case 'sanitizedProviderRecommendedGasPrice':
+      return sanitizedProviderRecommendedGasPrice(provider, gasOracleConfig, startTime);
```

Comment:
For consistency, it would be better `sanitizeProviderRecommendedGasPrice` for function naming.

Reply:
`sanitizedProviderRecommendedGasPrice` is better

Reply:
I proposed this naming for function naming because all other functions start with the verb `fetch`. However having regard to your recommendation, it'd be okay to name function `fetchProviderRecommendedGasPriceSanitized`. But as you can see, naming for all strategies end with `GasPrice` so  `fetchSanitizedProviderRecommendedGasPrice` is also good option.

Reply:
Sorry, I thought the comment was for line 284 and not 285. In that case, yes, it should be `fetchSanitizedProviderRecommendedGasPrice()` for consistency.

`fetchProviderRecommendedGasPriceSanitized()` only makes sense if the strategy was named `providerRecommendedGasPriceSanitized`, which I was suggesting above, but thinking about it again, it doesn't sound right.